src/main/java/com/saralein/server/response/ResponseHandler.java

```diff
@@ -0,0 +1,12 @@
+package com.saralein.server.response;
+
+import com.saralein.server.connection.Connection;
+
+import java.io.IOException;
+
+public class ResponseHandler {
```

Comment:
To you, what is a "Handler"? So far there is a `ResponseHandler` and a `ConnectionHandler`

Reply:
Thinking about this, `ResponseHandler` is gone now, but perhaps was misnamed.  In the case of `ConnectionHandler`, I was thinking of `Handler` in a sense like, we have this event which returns a `Connection` when the server socket receives a client request.  So the `ConnectionHandler` is the handler for that event, which will take care of the normal request/response operation with connected sockets.

I am also finding it hard to name some of these things.